test/unit/request.test.js

```diff
@@ -581,6 +592,56 @@ describe('Request', function () {
                 ]
             });
         });
+
+        it('should delete auth request when type is null', function () {
+            var rawRequest = {
+                    url: 'postman-echo.com',
+                    method: 'GET',
+                    header: [
+                        {
+                            key: 'some',
+                            value: 'header'
+                        },
+                        {
+                            key: 'other',
+                            value: 'otherheader',
+                            disabled: true
+                        }
+                    ],
+                    auth: null
```

Comment:
set a valid auth

Reply:
The test is for auth request when the type is null